Loop/View Controllers/SettingsTableViewController.swift

```diff
@@ -189,6 +190,17 @@ final class SettingsTableViewController: UITableViewController, DailyValueSchedu
                 return switchCell
             }
 
+            if case .fetchPumpGlucose = ConfigurationRow(rawValue: indexPath.row)! {
+                let switchCell = tableView.dequeueReusableCell(withIdentifier: SwitchTableViewCell.className, for: indexPath) as! SwitchTableViewCell
+
+                switchCell.`switch`?.isOn = dataManager.fetchPumpGlucoseEnabled
+                switchCell.titleLabel.text = NSLocalizedString("Fetch Pump Glucose", comment: "The title text for the fetch pump glocose enabled switch cell")
```

Comment:
Similar to what @bharat suggested, 'Fetch Enlite Data' would probably be better here.